src/App.tsx

```diff
@@ -1,10 +1,80 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useEffect,
+  useState,
+  useMemo,
+  useCallback,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Sort } from './utils/Sort';
+import { ErrorType } from './utils/ErrorType';
+import { TodosList } from './components/TodosList';
+import { Footer } from './components/Footer';
+import { Error } from './components/Error';
 
-const USER_ID = 0;
+const USER_ID = 10360;
 
 export const App: React.FC = () => {
+  const [newTodo, setNewTodo] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [sortType, setSortType] = useState<Sort>(Sort.All);
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.None);
+
+  const loadTodos = async () => {
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setTodos(todosFromServer);
+    } catch {
+      setErrorType(ErrorType.Add);
+      setTimeout(() => {
+        setErrorType(ErrorType.None);
+      }, 3000);
+    }
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
+  const sortingTodos = (type: Sort) => {
```

Comment:
```suggestion
  const sortTodos = (type: Sort) => {
```
It is a good tone to name functions according to these rules
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/96409563/723c4ec8-19cb-4d37-a4b6-26db63bfe562)